### DISCUSSION TITLE
Improve Markdown Heading rendering

### DISCUSSION THREAD

**Discussion by jasonwilliams** [GitJournal/GitJournal#791] [Feature Requests]
I'm writing markdown files then viewing them in GitJournal, but the layout could be improved compared to other Markdown readers. The headings are too close together with the text making the page look quite clunky.

**Here is some markdown on VSCode:**
![image](https://user-images.githubusercontent.com/936006/87075673-9e5b2b00-c218-11ea-9b8a-5bfd12cda594.png)
To be honest i think VSCode have too much spacing on either side and should reduce the amount of spacing between the heading and the text below it.

**Evernote Rendering:**
![image](https://user-images.githubusercontent.com/936006/87076343-86d07200-c219-11ea-9f18-256bede285d6.png)

**Here is the same document on GitJournal:**
![IMG_0012](https://user-images.githubusercontent.com/936006/87075693-a7e49300-c218-11ea-9fc6-e14dff6481f3.PNG)

**Expected Outcome**

It would be nice if the headings had more spacing above them, so it's more broken down into sections. Right now it all looks too close together.

Evernote would be a good example to copy

**Comment by vHanda** [comment]
I love that you have presented clear examples on how to improve it. Thanks a lot. This makes it so much easier to implement.

I also like the look of Evernote more. I'm going to try and copy it!

**Comment by jasonwilliams** [comment]
Thanks @vHanda thats great news!

If it helps “My main title here” is a h1.
“Subtitle” and “Date” are H2s, “Authors” and “another subheading” are H3s, “first”, “second” h4s

This may have been obvious from the image but was worth pointing out.

It would be nice if the h1 was just the title but that’s related to: https://github.com/GitJournal/GitJournal/issues/112

**Comment by jasonwilliams** [comment]
related: https://github.com/microsoft/vscode/issues/102036

**Comment by vHanda** [comment]
Hey Jason. Do you think we should increase the spacing on the left and right as well? It's currently 16px in order to align it with the 'x' and follow the material design specs.

**Comment by jasonwilliams** [comment]
The left and right spacing looks ok to me, so i would leave that for now. 
We can take another look at it once the headings spacing is done

**Comment by jasonwilliams** [comment]
How's this going @vHanda ?
Do you need any help?

**Comment by jasonwilliams** [comment]
I’ve noticed h1s are going into the right place now, awesome.
Quick note, h1s (or the journal title) and h2s seem to be the same size, not sure if you’re already aware.

**Comment by jasonwilliams** [comment]
Is there any update on this @vHanda ?
